bsip-0043.md

```diff
@@ -15,99 +15,42 @@ When creating a new asset, the asset owner is the only beneficiary of the market
 
 However, there might be another opportunity to promote the asset and stimulate the trading - use native Bitshares referral program. At this time unfortunately an assets owner is not able to share market fees with registrars and referrers to stimulate them to market the asset trading, so we suggest to add this possibility. Furthermore, enabling this feature for MPAs (e.g. bitCNY or bitUSD) can provide additional bounty for Bitshares registrars and referrals which can lead to more traders joining to the ecosystem.
 
-The workflow may be as follows. It is copied from vesting balance mechanism and got simplified.
+An asset owner defines the **market_fee_reward_percent**  in asset options - what percentage of the market fee he wants to share with the registrar.
+**market_fee** * **market_fee_reward_percent** goes the registar's fee pool.
+**market_fee** * (1 - **market_fee_reward_percent**) goes to the asset owner.
```

Comment:
1. Although I've read below that referrers will get a cut of the reward, words here seems only registrar will get rewarded;
2. registrar is an account which doesn't have a fee pool. Asset has a fee pool.